cartridge-cli.lua

```diff
@@ -3327,6 +3348,36 @@ local function remove_build_dir()
     end
 end
 
+local function detect_sdk_path(args)
+    -- check that passed one option for SDK
+    local sdk_params_are_right = check_that_only_one_is_true({
```

Comment:
Looks like an overkill - to write separate function to check that array of two booleans contains only one "true" value.

Reply:
It was written when there was 3 parameters.

I wouldn't write it for 2 parameters, but it's already written.